src/hooks/useViolations.ts

```diff
@@ -47,11 +43,17 @@ const violationFields: Record<ViolationName, ViolationField> = {
 
 type ViolationsMap = Map<ViolationField, TransactionViolation[]>;
 
+/**
+ * Hook to access violations for a transaction. Returns `getViolationsForField()`
+ * @example const {getViolationsForField} = useViolations(transactionViolations);
+ * @param violations - Array of {@link TransactionViolation}s
+ * @returns - Object with `getViolationsForField()` callback
+ */
 function useViolations(violations: TransactionViolation[]) {
     const violationsByField = useMemo((): ViolationsMap => {
         const violationGroups = new Map<ViolationField, TransactionViolation[]>();
 
-        for (const violation of violations) {
+        for (const violation of violations ?? []) {
```

Comment:
What's this doing here?

Reply:
In the vein of "I haven't seen this before", not "How could you dare put this here"

Reply:
I added that because I was getting an error that it could possibly be null. But looking at it, it doesn't seem like it should. Let me see if it's a weird linter rule or what.